### PR TITLE
feat: Added sync schedule settings fields

### PR DESCRIPTION
**Pull request by stark-chase**

### CODE REVIEW

File: manifest/manifest.schema.json [hunk 1]
```diff
@@ -332,6 +332,12 @@
             ],
             "pattern": "^(.*)$"
           },
+          "type": {
+            "$id": "#/properties/sync/properties/type",
+            "type": "string",
+            "title": "sync type",
+			"enum": ["Snapshot", "Incremental"]
```
Comment (omer-chase) [inline manifest/manifest.schema.json:339]: I prefer lower case

File: manifest/manifest.schema.json [hunk 2]
```diff
@@ -332,6 +332,12 @@
...
+          },
           "schedule": {
```
Comment (omer-chase) [inline manifest/manifest.schema.json:341]: use a `pattern` to validate the input

Comment (stark-chase) [reply manifest/manifest.schema.json:341]: No need for pattern - the enum validates if the value equals to one of the array's values

Comment (omer-chase) [reply manifest/manifest.schema.json:341]: i meant pattern for the `schedule`

**Pull request merged**